Source/Core/Core/ConfigManager.cpp

```diff
@@ -591,6 +594,15 @@ void SConfig::LoadCoreSettings(IniFile& ini)
 	core->Get("RSHACK", &bRSHACK, false);
 	core->Get("Latency", &iLatency, 2);
 	core->Get("SlippiOnlineDelay", &m_slippiOnlineDelay, 2);
+	core->Get("SlippiSaveReplays", &m_slippiSaveReplays, true);
+	core->Get("SlippiReplayMonthFolders", &m_slippiReplayMonthFolders, false);
+	#ifdef _WIN32
+		core->Get("SlippiReplayDir", &m_strSlippiReplayDir,
+			File::GetExeDirectory() + "\\" + "Slippi");
+	#else
+		core->Get("SlippiReplayDir", &m_strSlippiReplayDir,
+			File::GetExeDirectory() + DIR_SEP + "Slippi");
```

Comment:
`GetExeDirectory()` won't work on macOS, so this probably needs an `__APPLE__` (or OSX) check. I tested locally with the following and things work:

- Replace this with `File::GetHomeDirectory() + DIR_SEP + "Slippi"`.
- Add `GetHomeDirectory()` in `Source/Core/Common/FileUtil', where `GetHomeDirectory()` looks like:

```
std::string GetHomeDirectory()
{
    const char* home = getenv("HOME");
    return std::string(home);
}
```

This will make replays save by default in `/Users/<username>/Slippi`.

Reply:
Updated it to use GetBundleDirectory() at @NikhilNarayana 's suggestion on Discord

Reply:
Eeeehh I really don't think `GetBundleDirectory()` is good for this, as most macOS users don't see an app bundle as a folder. Accessing that location requires a somewhat cumbersome right click -> show contents that feels like it'd require handholding.

Reply:
I should have clarified: it uses a folder next to the app bundle, not the bundle folder itself, which is what Slippi r18 did apparently.